src/lib/coda_base/tick_intf.ml

```diff
@@ -0,0 +1,27 @@
+open Core
+
+module type TICK = sig
```

Comment:
For `Non_zero_curve_point`, I created an additional layer, with `Tick` inside, because the contained modules were mutually recursive.

Reply:
Which turned out not to be needed.